### DISCUSSION TITLE
A bug with an academic proof upload

### DISCUSSION THREAD

**Discussion by fatrybl** [community/community#86726] [GitHub Education]
### Hi everyone,

Hello everyone, i was trying to submit my application for Education program but faced an issue:
When i tried 1-st time and uploaded a proof. then pressed "Submit" button, it was said that i have to enable 2FA authentication and fill in billing info in my account. 
I did it.
Then i tried to submit a proof 2-nd time with the previous issues being fixed, but it says: 
    Your application cannot be reviewed until you fix the following:
        Your uploaded proof has already been used previously.
So i can not upload the same proof. I`m stucked here. I tried to log out and restart web-browser, did not help.
I dont know if my application is being checked or even have not been submitted.
![image](https://github.com/community/community/assets/22779281/fcf959cc-f988-4bae-8a1f-b39b50b908d7)

**Comment by fsollecito** [comment]
Same exact thing happened to me.  Now I'm stuck.  Don't see any way to contact someone at GitHub, just keeps routing me back to this forum.  Very frustrating.
